src/Bridge/Symfony/Bundle/Resources/config/swagger.xml

```diff
@@ -13,7 +13,7 @@
             <argument type="service" id="api_platform.metadata.property.metadata_factory" />
             <argument type="service" id="api_platform.resource_class_resolver" />
             <argument type="service" id="api_platform.operation_method_resolver" />
-            <argument type="service" id="api_platform.iri_converter" />
+            <argument type="service" id="api_platform.path_resolver.operation" />
```

Comment:
Shouldn't the service be named `api_platform.operation_path_resolver` for consistency with other services?